### PR TITLE
Add icon

### PR DESCRIPTION
**Pull request by ramidzkh**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const electron = require('electron');
-const {app, BrowserWindow} = electron;
+const {app, BrowserWindow, NativeImage} = electron;
```
Comment (hYdos) [inline index.js]: remove unneeded NativeImage

**Review by hYdos — CHANGES_REQUESTED** [review]
please see comments

File: index.js (hunk 1 above)
Comment (ramidzkh) [reply index.js]: outdated

**Review by hYdos — APPROVED** [review]
i approv

**Pull request merged**